src/set_theory/cofinality.lean

```diff
@@ -495,6 +495,62 @@ theorem succ_is_regular {c : cardinal.{u}} (h : omega ≤ c) : is_regular (succ
     apply typein_lt_type }
 end⟩
 
+/--
+A useful special case of the pigeonhole principal with `θ = (mk α).succ`.
+
+A function whose codomain's cardinality is infinite but strictly smaller than its domain's
+has a fiber with cardinality strictly great than the codomain.
+-/
+theorem infinite_pigeonhole' {β α : Type u} (f : β → α)
```

Comment:
To match with `infinite_pigeonhole_card`, how about:
```suggestion
theorem infinite_pigeonhole_card_lt {β α : Type u} (f : β → α)
```

Reply:
Good!